src/java.desktop/share/classes/java/awt/image/Raster.java

```diff
@@ -369,7 +369,6 @@ public static WritableRaster createBandedRaster(int dataType,
         int[] bandOffsets = new int[bands];
         for (int i = 0; i < bands; i++) {
             bankIndices[i] = i;
-            bandOffsets[i] = 0;
```

Comment:
I suggest to comment this line and adding a comment like "0 zero is the default value not need to be set"

Reply:
I'm not really fan of commented out code.
What if I add a comment to the array declaration itself?
```
int[] bandOffsets = new int[bands]; // by default have 0 values
```

Reply:
Yeah, please, no commented-out code. I'd be fine with the code as it was, I don't think it needs much clarification. If @mrserb insists on a comment then I agree that a comment on the array declaration should be sufficient.

Reply:
That is not about the default values in the array but about missing initialization of `bandOffsets`, which we will skip intentionally. So it won't raise the question later why we initiated the Indices and not offsets.

Reply:
I've added comment and moved array declaration. I think currently code is clear enough.

Reply:
Yeah, the message is super clear now.